src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -58,6 +58,10 @@ public static UserStore getTestInstance(PersistentStorageAgent persistentStorage
   /** The in-memory list of Users. */
   private List<User> users;
 
+  public List<User> getAllUsers() {
+        return users;
```

Comment:
nit: Can you fix this indentation? Should be 2 spaces for top level and 4 for second level.

Reply:
Good habit to review your PR and diffs before adding reviewers so that small changes like these which can be difficult to catch in a text editor are cleaned up